pcsx2/MemoryTypes.h

```diff
@@ -18,6 +19,8 @@ namespace Ps2MemSize
 	static constexpr u32 IopHardware = _64kb;
 
 	static constexpr u32 GSregs = 0x00002000; // 8k for the GS registers and stuff.
+
+	extern u32 GetExposedRam();
```

Comment:
Rather than a function, and conditional move on loading a config variable, just have a new global that contains the memory size. Since it has to do a memory load anyway, this is free, and avoids the cmove.

Reply:
Would you prefer a mutable global:
```cpp
static u32 ExposedRam = MainRam;
```
Or an `ExposedRam()` function that returns a private global to prevent outside modification? I assume the former given this suggestion came after the request to simply rename `GetExposedRam()`.

Reply:
Global variable would be better. Debug builds are already slow enough without inlining, no need to make them slower.